Updating RHBQ procedure (#128)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc

    ...
</project>
----
+
. Remove the `quarkus-kogito-bom` dependency from the `pom.xml` file of your project:
+
.Dependencies in `pom.xml` file
[source,xml,subs="attributes+"]
----
<project>
    ...
    <dependencyManagement>
        <dependencies>
            ...
            <dependency>
                <groupId>io.quarkus.platform</groupId>
                <artifactId>quarkus-kogito-bom</artifactId>

>                 <version>2.7.6.Final</version>

Reviewer: If they follow the steps, this will be 2.7.6.Final-redhat-00006.  At least currently, but it will automatically be updated if RHBQ does a new 2.7.6.x release.

                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
    ...
</project>
----
+

> . Follow the instructions in the link:{redhat_registry_auth_url}[Red Hat Container Registry Authentication] article to login to the registry an use the OpenShift Serverless Logic images locally.

Reviewer 2: Do we need to use the attribute instead of OpenShift Serverless Logic?
```suggestion
. Follow the instructions in the link:{redhat_registry_auth_url}[Red Hat Container Registry Authentication] article to log in to the registry and use the OpenShift Serverless Logic images locally.
```

Author: Yes! I'll change that after merging your PR then.
